packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -240,6 +244,14 @@
   "Settings.error": "Error",
   "Settings.permissions.menu.link.roles.label": "Roles",
   "Settings.permissions.menu.link.users.label": "Users",
+  "Settings.permissions.users.add-new": "Add new user",
+  "Settings.permissions.users.create": "Create new User",
+  "Settings.permissions.users.form.email": "Email",
+  "Settings.permissions.users.form.firstname": "First name",
+  "Settings.permissions.users.form.lastname": "Last name",
+  "Settings.permissions.users.listview.header.title": "Users",
+  "Settings.permissions.users.listview.header.description.singular": "0 users found",
```

Comment:
This singular message will only be used for the `0 users found`? 
Don't we need a `1 user found`?

Reply:
You are right!